spec/classes/letsencrypt_scripts_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'letsencrypt::scripts' do
+  on_supported_os.each do |os, facts|
+    let :facts do
+      facts
+    end
```

Comment:
```suggestion
  on_supported_os.each do |os, os_facts|
    let :facts do
      os_facts
    end
```

that makes it easier to not mix up facts from on_supported_os.each and facts from the let block